support-payment-api/build.sbt

```diff
@@ -14,6 +14,7 @@ val circeVersion = "0.11.1"
 val AWSJavaSDKVersion = "1.11.568"
 val jacksonVersion = "2.10.0"
 val stripeVersion = "10.12.0" // Supports API version 2019-05-16
+val playCirceVersion = "2712.0"
```

Comment:
you can import project/LibraryVersions.scala from here as it's in the top level build, to reduce the risk

Reply:
Yes, I spotted that too earlier - I guess I wanted to get the upgrade bit tested first. Will put this in

Reply:
Done